content/documentation/edge/installation-guide/index.adoc

```diff
@@ -1,8 +1,9 @@
 ---
-title: "Getting started"
+title: "Installation guide"
 aliases:
+- /installation-guide/
```

Comment:
Remember that this gets versioned, so after the next release there's a chance those are going to break. I'm not exactly sure how hugo handle more than one page with the same alias, this seems to be working now but it you plan to depend on this link, that's something you need to check.

Reply:
TBH, I am no that familiar with the versioning system. I just replicated what it was before in the getting-started.
Is it worth it to add this new file under both `latest` and `edge`?

Reply:
latest is just a symbolic link to the latest released version, so the links could point there. "edge" means "latest master".

Reply:
an aside, but I still don't care for 'edge'.  I would have preferred 'staging' or even 'master'.
.

Reply:
@jshaughn I'm not married to the name at all, feel free to change :)

Reply:
@cfcosta this aliases were already there before this PR. It looks like when there are duplicates, hugo takes the latest release. I am not 100% why takes this release and not another one. I've tried to add a v1.20, v1.21, v1.30 and in all the cases it has always taken the highest release, so it seams quite safe.

See the example: kiali.io/gettingstarted (been there since first release)

Reply:
I'm pleasantly surprised :P Was not expecting it to work, but if it does, who am I to complain? :)